ProductionModePlugin.js

```diff
@@ -67,34 +67,37 @@ class ProductionModePlugin {
         if(param.isString() && param.string === "globalize" && this.moduleFilter(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
 
-          // Statically extract Globalize formatters and parsers from the request
-          // file only. Then, create a custom precompiled formatters/parsers module
-          // that will be called instead of Globalize, which in turn requires
-          // Globalize, set the default locale and then exports the Globalize
-          // object.
-          const compiledDataFilepath = globalizeCompilerHelper.createCompiledDataModule(request);
+          this.supportedLocales.forEach((locale) => {
```

Comment:
@TheLarkInn, is there a way to create virtual dependency (all in memory) in webpack? Currently, we're dynamically creating a file, then using `CommonJsRequireDependency` and passing it the filepath.